neural_compressor/adaptor/ox_utils/calibration.py

```diff
@@ -443,7 +467,7 @@ def calculate_quantization_params(self, q_config, quantization_thresholds):
 
         return quantization_params
 
-    def dump_tensor(self, activation=True, weight=False):
+    def dump_tensor(self, q_config=None, activation=True, weight=False):
```

Comment:
dump_tensor function is used to dump tensor only, add q_config parameter is unreasonable